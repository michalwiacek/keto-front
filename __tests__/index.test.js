/* eslint-env jest */

import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import IndexPage from '../pages/index.tsx';

test('index page loads normally', async () => {
  const { getByText, findByText } = render(<IndexPage />);

  expect(getByText('Get to where you want to be.')).toBeInTheDocument();
});
