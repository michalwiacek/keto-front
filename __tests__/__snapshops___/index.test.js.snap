exports[`With React Testing Library Snapshot Should match Snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Hello World!
    </p>
  </div>
</DocumentFragment>
`;